PyInstaller/building/osx.py

```diff
@@ -163,6 +168,8 @@ def assemble(self):
         links = []
         toc = add_suffix_to_extensions(self.toc)
         for inm, fnm, typ in toc:
+            if typ == 'OPTION':
+                continue
```

Comment:
This is an unrelated change and shall go into a commit of it's own.

Reply:
I had to add this to prevent such exception:
```python
8090 INFO: Building BUNDLE BUNDLE-00.toc
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "pyinstaller/pyinstaller.py", line 15, in <module>
    run()
  File "pyinstaller/PyInstaller/__main__.py", line 109, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "pyinstaller/PyInstaller/__main__.py", line 61, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "pyinstaller/PyInstaller/building/build_main.py", line 838, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "pyinstaller/PyInstaller/building/build_main.py", line 784, in build
    exec(text, spec_namespace)
  File "<string>", line 45, in <module>
  File "pyinstaller/PyInstaller/building/osx.py", line 91, in __init__
    self.__postinit__()
  File "pyinstaller/PyInstaller/building/datastruct.py", line 158, in __postinit__
    self.assemble()
  File "pyinstaller/PyInstaller/building/osx.py", line 182, in assemble
    shutil.copy(fnm, tofnm)
  File "/lib/python3.6/shutil.py", line 241, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: ''
```
But I will remove it because it will become useless after the next round.